src/modules/bot/bot.service.spec.ts

```diff
@@ -735,4 +735,13 @@ describe('BotService', () => {
     deletedIds = [];
     fetchMock.restore();
   });
+  it('should return bot IDs in the response', async () => {
+    fetchMock.delete(`${configService.get<string>('UCI_CORE_BASE_URL')}${configService.get<string>('CAFFINE_INVALIDATE_ENDPOINT')}`,
+    true
+    );
+    const response = await botService.remove({ids: ['testId1', 'testId2'], endDate: null});
```

Comment:
You also need to disable the status of bot, since only then the bots will be deleted.